Modules/pyexpat.c

```diff
@@ -703,6 +703,37 @@ get_parse_result(pyexpat_state *state, xmlparseobject *self, int rv)
 
 #define MAX_CHUNK_SIZE (1 << 20)
 
+/*[clinic input]
+pyexpat.xmlparser.SetReparseDeferralEnabled
+
+    cls: defining_class
```

Comment:
`cls` is not used, it is better to remove it.

Reply:
@serhiy-storchaka good point, will be gone with the next push in a minute.